src/ReceivingList/ReceivingList.js

```diff
@@ -51,6 +54,16 @@ const ReceivingList = ({
     changeSorting,
   ] = useLocationSorting(location, history, resetData, sortableFields);
 
+  const selectedTitle = useCallback(
+    (e, { id }) => {
+      history.push({
+        pathname: `/receiving/${id}/view`,
+        search: location.search,
```

Comment:
is there any reason to do so instead of ```history.push(`/receiving/${id}/view`)``` ?

Reply:
to keep checked filters